src/System.Resources.Extensions/src/System/Resources/Extensions/SerializationFormat.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Resources.Extensions
+{
+    // Internal Enum that's shared between reader and writer to indicate the 
+    // deserialization method for a resource.
+    internal enum SerializationFormat : byte
```

Comment:
>byte [](start = 40, length = 4)

why byte?

Reply:
Because I want it to take up less space in the .resources file and we're unlikely to ever need more than 255 values for serialization methods.  I suppose I could leave it be and write it as a 7bit-encoded int.  Let me do that.